app/queries/my_facilities/blood_pressure_control_query.rb

```diff
@@ -32,15 +32,20 @@ def cohort_uncontrolled_bps
     @cohort_period == :month ? monthly_uncontrolled_bps : quarterly_uncontrolled_bps
   end
 
-  def all_time_bps
-    @all_time_bps ||= LatestBloodPressuresPerPatient
-                      .where('patient_recorded_at < ?', Time.current.beginning_of_day - 2.months)
-                      .where(bp_facility_id: facilities)
+  def cohort_bps
+    @cohort_period == :month ? monthly_bps_cte : quarterly_bps
+  end
+
+  def all_time_patients
+    @all_time_patients ||= Patient
+                      .where('recorded_at < ?', Time.current.beginning_of_day - 2.months)
```

Comment:
Just clarifying, since we're using the `Patient` table directly now, we now include patients with 0 BPs in the denominator (as intended), right?

Reply:
I removed the 2.months constant since we've stopped using it, but I can add it back for this query.

`patients.recorded_at` is either the time at which the patient record was created on the app or the `recorded_at` of their earliest BP, whichever is earlier (this is to support retroactive data entry).

Yes, we are now including patients with 0 BPs in the denominator.

Reply:
Ah, `recorded_at` is the _earliest_ activity. Nice.